docs/howto_config.md

```diff
@@ -75,51 +75,57 @@ The list of manifest locations can be customized via configuration using the abo
 
 ## Manifest Example
 
-The example demonstrates automatic deployment of an `nginx` container using `kustomize` manifests in the `/etc/microshift/manifests` directory.
+The example demonstrates automatic deployment of a `busybox` container using `kustomize` manifests in the `/etc/microshift/manifests` directory.
 
 Run the following command to create the manifest files.
 
 ```bash
 MANIFEST_DIR=/etc/microshift/manifests
 sudo mkdir -p ${MANIFEST_DIR}
 
-sudo tee ${MANIFEST_DIR}/nginx.yaml <<EOF
+sudo tee ${MANIFEST_DIR}/busybox.yaml &>/dev/null <<EOF
+apiVersion: v1
+kind: Namespace
+metadata:
+  name: busybox
+---
 apiVersion: apps/v1
 kind: Deployment
 metadata:
-  name: nginx-deployment
+  name: busybox-deployment
 spec:
   selector:
     matchLabels:
-      app: nginx
-  replicas: 2
+      app: busybox
   template:
     metadata:
       labels:
-        app: nginx
+        app: busybox
     spec:
       containers:
-      - name: nginx
-        image: NGINX_IMAGE
-        ports:
-        - containerPort: 8080
+      - name: busybox
+        image: BUSYBOX_IMAGE
+        command:
+          - sleep
+          - "3600"
```

Comment:
I'm checking if something like this would work with this image
```suggestion
    - name: hello-world-http
      image: busybox
      command: ['sh','-c', 'echo "hello world" > index.html && /bin/httpd -p 9000 -f']
      ports:                 # A port this container is exposing
      - containerPort: 9000
        protocol: TCP
```